src/components/Answer.js

```diff
@@ -0,0 +1,28 @@
+import React, {useState}from "react";
+
+
+const Answer = (answerObject) => {
```

Comment:
```suggestion
const Answer = (data) => {
```

Reply:
Doordat het al in de component Answer staat hoeft er niet specifiek aangegeven te worden watvoor object het is. Je kan in dit soort gevallen beter generale namen kiezen